install_nvm.sh

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env bash
```

Comment:
Why are you adding this file, this is your system specific file i suppose . Revert this commit changes